flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
Fedora builds it for i386 too.

Reply:
The build fails in the pyside module for i386, arm and aarch64 with same error:

https://flathub.org/builds/#/builders/2/builds/5530
```
  :0: error: Unable to retrieve code snippet. [other]
  :0: error: Unable to retrieve code snippet. [other]
  :0: error: Unable to retrieve code snippet. [other]
  :0: error: Unable to retrieve code snippet. [other]
  :0: error: Unable to retrieve code snippet. [other]
  :0: error: Unable to retrieve code snippet. [other]
  :0: error: Unable to retrieve code snippet. [other]
  :0: error: Unable to retrieve code snippet. [other]
Keeping temporary file: /tmp/QtCore_global_RbFMFE.hpp
shiboken: Error running ApiExtractor.
Command line: --generator-set=shiboken --enable-parent-ctor-heuristic --enable-pyside-extensions --enable-return-value-heuristic --use-isnull-as-nb_nonzero /run/build/pyside2/sources/pyside2/_build/PySide2/QtCore_global.h --include-paths=/app/qt5include:/usr/include/c++/6.4.0:/usr/include/c++/6.4.0/x86_64-unknown-linux:/run/build/pyside2/sources/pyside2/PySide2:/usr/include/ --typesystem-paths=/run/build/pyside2/sources/pyside2/_build/PySide2:/run/build/pyside2/sources/pyside2/PySide2:/run/build/pyside2/sources/pyside2/PySide2/QtCore --output-directory=/run/build/pyside2/sources/pyside2/_build/PySide2/QtCore --license-file=/run/build/pyside2/sources/pyside2/PySide2/QtCore/../licensecomment.txt /run/build/pyside2/sources/pyside2/_build/PySide2/QtCore/typesystem_core.xml --api-version=5.11 --drop-type-entries=
ninja: build stopped: subcommand failed.
]2;flatpak-builder: Cleanup pyside2Error: module pyside2: Child process exited with code 1
FB: unmounting rofiles-fuse /srv/buildbot/worker/build-i386-2/org.freecadweb.FreeCAD/.flatpak-builder/rofiles/rofiles-LsMr4v
program finished with exit code 1
elapsedTime=745.785434
```

If you can point me to a fix, I can enable the other arches.